app/helpers/schema_helper.py

```diff
@@ -52,6 +52,11 @@ def get_repeat_rule(group):
                 if 'repeat' in rule.keys():
                     return rule['repeat']
 
+    @staticmethod
+    def get_skip_condition(group):
+        if 'skip_condition' in group:
```

Comment:
May as well `return group.get('skip_condition')` which either returns the value if found or None.